### ISSUE TITLE
Add a CLI tool to query a TSDB block

### ISSUE THREAD

**Issue by fpetkovski** [thanos-community/promql-engine#269]
It would be great to have something like `promql-cli` that allows executing a query against a TSDB block. We can also have a flag which toggles the creation of CPU and heap profiles for the query execution.

**Comment by yeya24** [comment]
Is this feature in the scope of this repo?

**Comment by fpetkovski** [comment]
My thought was that it can be useful for benchmarking and profiling queries. The CLI tool could output time spent in each operator, memory used by operator etc. So it should be more than a generic querying tool.

**Comment by yeya24** [comment]
> The CLI tool could output time spent in each operator, memory used by operator etc

That's an interesting idea. Do we have those metrics built in this library now? We don't have a way to track time and memory usage now. Shouldn't we have that first before the CLI?

**Comment by fpetkovski** [comment]
Agreed, we would need https://github.com/thanos-io/promql-engine/issues/106 to be done first.
